teku/src/main/java/tech/pegasys/teku/cli/options/MetricsOptions.java

```diff
@@ -71,19 +75,19 @@
   private int idleTimeoutSeconds = MetricsConfig.DEFAULT_IDLE_TIMEOUT_SECONDS;
 
   @Option(
-      names = {"--Xmetrics-endpoint"},
+      names = {"--Xmetrics-publish-endpoint"},
       hidden = true,
       paramLabel = "<ENDPOINT>",
```

Comment:
I wonder if this should be `<URL>` - `<ENDPOINT>` isn't really a type.

Reply:
yep sure, can change that